src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -66,6 +64,17 @@ else if (element instanceof HaxeVarDeclaration) {
   }
 }
 
+class ImportChecker {
+  public static void check(HaxeImportStatementRegular element, AnnotationHolder holder) {
+    HaxeImportModel importModel = element.getModel();
+    HaxeClassModel aClass = importModel.getHaxeClass();
+    if (aClass == null) {
+      Annotation annotation = holder.createErrorAnnotation(element, "Can't find class");
+      annotation.registerFix(new HaxeRemoveFixer(element));
```

Comment:
I guess, nothing.  The only thing that I could see at the moment would be to offer classes with similar names, like a spell checker, but we already have completion.  

I don't know why I thought that 'add import' would be useful.